lib/generators/rolemodel/optics/base/base_generator.rb

```diff
@@ -0,0 +1,35 @@
+require_relative '../../../bundler_helpers'
+
+module Rolemodel
+  module Optics
+    class BaseGenerator < Rails::Generators::Base
+      include Rolemodel::BundlerHelpers
+      source_root File.expand_path('templates', __dir__)
+
+      def add_optics_package
+        run 'yarn add @rolemodel/optics'
+      end
+
+      def add_slim
+        gem 'slim'
```

Comment:
Does it still make sense for this to be mixed in with Optics? Could we separate it?